s3_transport/include/irods/private/s3_transport/s3_transport.hpp

```diff
@@ -1551,8 +1567,11 @@ namespace irods::experimental::io::s3_transport
 
                 // set the number of threads to close to the number of client transfer threads
                 // this will count down as closes happen
+//std::to_string(std::hash<std::string>{}(config_.resource_name + "/" + object_key_));
+logger::debug("{}:{} ({}) [[{}]] threads_remaining_to_close in shmem is  {}, shmem_key_={}, resource_name={}, object_key_={}", __FILE__, __LINE__, __FUNCTION__, get_thread_identifier(), data.threads_remaining_to_close, this->shmem_key_, this->config_.resource_name, this->object_key_);
```

Comment:
```suggestion
logger::debug("{}:{} ({}) [[{}]] threads_remaining_to_close in shmem is {}, shmem_key_={}, resource_name={}, object_key_={}", __FILE__, __LINE__, __FUNCTION__, get_thread_identifier(), data.threads_remaining_to_close, this->shmem_key_, this->config_.resource_name, this->object_key_);
```

Reply:
These were just debug messages with the plan of removing this.  I should have changed this pull request to in progress so it wouldn't be reviewed.  These have now been removed.